backend/src/test/java/de/neuefische/teddyscoronadiaries/controller/WeatherControllerTest.java

```diff
@@ -70,11 +68,11 @@ public void getProvinceCapitalWeatherDataShouldReturnWeatherData() {
         assertThat(response.getStatusCode(), is(HttpStatus.OK));
         assertThat(response.getBody(), is(new ProvinceCapitalWeatherData(
```

Comment:
Using a builder here helps me understand what's going on. So this seems to be a DTO maybe rename to `ProvinceCapitalWeatherDto` ?